panel/models/perspective.ts

```diff
@@ -72,82 +72,64 @@ export class PerspectiveView extends HTMLBoxView {
     this.connect(this.model.source.properties.data.change, () => this.setData())
     this.connect(this.model.source.streaming, () => this.stream())
     this.connect(this.model.source.patching, () => this.patch())
-    this.connect(this.model.properties.schema.change, () => {
+
+    const {
+      schema, toggle_config, columns, expressions, split_by, group_by,
+      aggregates, filters, sort, plugin, selectable, editable, theme,
+    } = this.model.properties
+
+    const not_updating = (fn: () => void) => {
```

Comment:
I think this is too much abstraction for my likening.

Reply:
This is no different from a simple decorator. Not only this allows to shorten the code, but more importantly, it semantically connects a collection of previously unrelated signal handlers.

Reply:
You are overengineering a simple if statement. Your change make the code harder to understand.

Reply:
On balance I'd agree with keeping the simpler version just because it's such a simple if condition and barely saves any lines of code. That said I don't feel particularly strongly either way.